app/assets/javascripts/backbone/views/calendar_view.coffee

```diff
@@ -1,6 +1,28 @@
 # Common functions for calendars application-wide.
 class MS.Views.CalendarView extends Backbone.View
 
+  events:
+    'click .cal-step': 'stepClick'
+
+  prepareVariables: (params) ->
+    @$calendar = @$('#calendar')
+    @stepModal = params.stepModal
```

Comment:
Both use it. This particular feature (adding ability to edit steps from dashboard) will need tge step modal.